src/types/composite/list.ts

```diff
@@ -376,4 +385,13 @@ export class CompositeListType<T extends List<object> = List<object>> extends Co
     this.tree_setLength(target, length - 1);
     return value as T[number];
   }
+  tree_getLeafGindices(target?: Tree, root: Gindex = BigInt(1)): Gindex[] {
+    if (!target) {
+      throw new Error("variable type requires tree argument to get leaves");
+    }
+    const gindices = super.tree_getLeafGindices(target, root);
+    // include the length chunk
+    gindices.push(concatGindices([root, BigInt(3)]));
```

Comment:
Why `3` here? Could you add a comment explaining

Reply:
The comment on the line above is an attempt to explain :sweat_smile: